TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -298,9 +298,18 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 		_mediaBrowserView: null,
 
 		_beforeMediaBrowserIsShown: function () {
-			var that = this;
+			var that = this,
+				value = null;
+
+			try {
+				value = JSON.parse(this.get('value'));
+			} catch (exception) {}
```

Comment:
@aertmann how about logging that to the JS console but still ignoring it? Actually, the user can't really do anything to solve the problem and might be better off without an exception.